app/ide-desktop/lib/dashboard/src/authentication/cognito.ts

```diff
@@ -367,19 +371,49 @@ export interface UserSession {
   readonly email: string
   /** User's access token, used to authenticate the user (e.g., when making API calls). */
   readonly accessToken: string
+  /** User's refresh token, used to refresh the access token when it expires. */
+  readonly refreshToken: string
+  /** URL to refresh the access token. */
+  readonly refreshUrl: string
+  /** Time when the access token will expire, date in ISO format(UTC). */
```

Comment:
missing space before "(UTC)".
also technically ISO is the organization, do you mean ISO 8601? (also technically we normally use RFC 3339 format. `toISOString()` should give a valid RFC 3339 string but we also have a `dateTime` module.)

Reply:
also be consistent when mentioning whether it is a time or a date (or perhaps it's a date *and* time?))

Reply:
toISOString returns ISO 8601
![CleanShot 2024-03-08 at 19 55 33@2x](https://github.com/enso-org/enso/assets/61194245/15a7d88c-dd28-4cf3-80c1-909055a127bf)

Reply:
But you've asked a good question...Will tomorrow reach Radek and ask him what format he'd like to work with

Reply:
ISO 8601 seems right, this is how we describe it in Enso

![image](https://github.com/enso-org/enso/assets/1436948/9de58625-76f6-4864-b056-4d75a3b61061)

Reply:
note that `toISOString()` format is compatible with both ISO8601 and RFC3339. cloud (and it seems rust in general) refers to it as RFC3339 <https://github.com/enso-org/cloud-v2/blob/4020438d22a596a0341cac1467564726bbb3dd16/src/lambdas/src/model/mod.rs#L987>

[relevant](https://ijmacd.github.io/rfc3339-iso8601/)